chain/jsonrpc-primitives/Cargo.toml

```diff
@@ -26,4 +27,5 @@ near-client-primitives = { path = "../client-primitives", optional = true }
 [features]
 full = ["debug_types"]
 debug_types = ["near-client-primitives"]
+arbitrary_feature = ["arbitrary"]
```

Comment:
`optional = true` should already create a `arbitrary` feature that you could use in the same way as `arbitrary_feature`, so I think you shouldn’t need this line